packages/blocks/src/api/utils.js

```diff
@@ -1,8 +1,10 @@
 /**
  * External dependencies
  */
-import { every, has, isFunction, isString, reduce } from 'lodash';
-import { default as tinycolor, mostReadable } from 'tinycolor2';
+import { every, has, isFunction, isString, reduce, maxBy } from 'lodash';
```

Comment:
Curious if this increase in lodash usage counterweights the benefits of colord

Reply:
According to our bot, it seems there is still a small reduction:
![image](https://user-images.githubusercontent.com/11271197/135062758-dfd41daf-3d49-4549-9a78-a71cd41a81e5.png)

But I think the main reason for the replacement is colord having first-party typescript types and being more performant in the benchmarks.

Reply:
Still good to keep an eye on if there was an intention to reduce the use of lodash

Reply:
I guess in this case we can easily avoid the maxBy and use a reduce. But maxBy makes the code more clear (it becomes easy to do what the function is doing) and it internally probably just uses a reduce anyway so the saving would not be noticeable.